arrayfire/library.py

```diff
@@ -422,6 +422,22 @@ class CANNY_THRESHOLD(_Enum):
     MANUAL = _Enum_Type(0)
     AUTO_OTSU = _Enum_Type(1)
 
+class FLUX(_Enum):
+    """
+    Flux function types
```

Comment:
instead of types, can we say something else ?

Reply:
@9prady9 Done.